Week02/lesson04/homework/relaxr_starter_code/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="description" content="Relaxr, the perfect personal assistant. Get Piece of mind with a single tap.">
+	<meta name="keywords" content="Relaxr, Personal Assistant, PA, AI Technology">
+	<title>Relaxr | The Perfect PA</title>
+
+	<!-- external CSS links -->
+	<link rel="stylesheet" href="css/reset.css">
+	<link rel="stylesheet" href="css/style.css">
+	<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,800" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+
+	<div id="container">
+		<header>
+			<h1>Relaxr</h1>
+			<h2>Get piece of mind with a single tap</h2>
+			<a href="http://www.relaxr.com/signup"><div id="button1">Get it Now</div></a>
+		</header>
+
+		<main>
+			<h3>Benefits</h3>
+			<p>The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
+				<ul>
+					<li>Schedule meetings for you</li>
```

Comment:
Not sure what is happening here as I already have all those settings as default? 🤔